composer.json

```
@@ -27,7 +27,8 @@
         "ibexa/core": "~4.3.0@dev",
         "ibexa/content-forms": "~4.3.0@dev",
         "ibexa/rest": "~4.3.0@dev",
-        "ibexa/http-cache": "~4.3.0@dev"
+        "ibexa/http-cache": "~4.3.0@dev",
+        "symfony/process": "*"
```

```suggestion
        "symfony/process": "^5.0"
```

and it should be moved so the list remains sorted alphabetically.